pkg/api/component_report.go

```diff
@@ -742,6 +742,10 @@ func (c *componentReportGenerator) normalizeProwJobName(prowName string) string
 	if prev, err := previousRelease(c.SampleRelease.Release); err == nil {
 		name = strings.ReplaceAll(name, prev, "X.X")
 	}
+	name = strings.ReplaceAll(name, "f28", "fXX")
+	name = strings.ReplaceAll(name, "f14", "fXX")
+	name = strings.ReplaceAll(name, "f7", "fXX")
+	name = strings.ReplaceAll(name, "f6", "fXX")
 	return name
```

Comment:
Will they have other frequencies?  I know you hate regex but this isn't too bad:

	re := regexp.MustCompile(`-f\d+`)
	output := re.ReplaceAllString(input, "-fXX")